stable/prometheus/values.yaml

```diff
@@ -638,6 +653,10 @@ pushgateway:
     #   cpu: 10m
     #   memory: 32Mi
 
+  ## Security context to be added to push-gateway pods
+  ##
+  securityContext: {}
```

Comment:
You forgot to add this one to the README :)

Reply:
Fixed with d400702, thx!